### PR TITLE
Link fixes5

### PR DESCRIPTION
**Pull request by TEChopra1000**
These fixes include the following.

- I added the data folder back into the Gluon tutorial folder.
- I fixed a few links in the gluon tutorial index page and the general tutorial index page
- As per Soji's request, I took the info_gan tutorial off of the Gluon tutorial index page. I will add the actual tutorial to the examples folder in my next pr. 
- I added the autograd images and videos back into the docs and corrected the references to them.

### CODE REVIEW

File: docs/python_docs/python/tutorials/packages/index.rst [hunk 1]
```diff
@@ -46,9 +46,9 @@ Shared APIs
 
    .. card::
       :title: Symbol API
-      :link: symbol/index.html
+      :link: /api/python/docs/api/symbol/index.html
```
Comment (larroy) [inline docs/python_docs/python/tutorials/packages/index.rst]: isn't it better that this link is relative too?

Comment (TEChopra1000) [reply docs/python_docs/python/tutorials/packages/index.rst]: @larroy yes, that's a good point. Thank you for catching that. I've updated this reference to a relative link.

**Comment by larroy** [comment]
There's something failing in the website build in CI

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Fwebsite/detail/PR-16975/4/pipeline

File: docs/python_docs/python/tutorials/packages/gluon/index.rst [hunk 2]
```diff
@@ -194,19 +194,15 @@ Advanced Topics
...
+      :link: loss/custom-loss.md
 
       A guide to implementing custom losses.
 
-..
-
    .. card::
       :title: Gotchas using NumPy in Apache MXNet
-      :link: ../ndarray/gotchas_numpy_in_mxnet.html
+      :link: ndarray/gotchas_numpy_in_mxnet.md
```
Comment (TEChopra1000) [inline docs/python_docs/python/tutorials/packages/gluon/index.rst]:
```suggestion
      :link: ../ndarray/gotchas_numpy_in_mxnet.md
```

**Pull request closed without merging**